x/genutil/client/cli/init.go

```diff
@@ -88,6 +88,13 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			config.Mempool.KeepInvalidTxsInCache = true
 			config.Mempool.TTLNumBlocks = 15
 
+			config.P2P.UseLegacy = true
+			config.P2P.MaxNumInboundPeers = 250
+			config.P2P.MaxNumOutboundPeers = 70
+			config.P2P.MaxConnections = 320
```

Comment:
Do these still reflect the latest thinking? I thought most validators decreased these values again. TBH, I am not sure we should hard-code these settings without very clear evidence that this is actually stabilizing the network.

Reply:
I agree. Also as suggested earlier and in #151, we should move this to the app